addons/mail/static/tests/messaging/messaging_tests.js

```diff
@@ -86,6 +75,7 @@ QUnit.test(
         await click(".o-mail-Composer-send:not(:disabled)");
         // leaving discuss.
         await openFormView("res.partner", partnerId);
-        assert.containsNone($, ".o-mail-ChatWindow-header:contains(Dumbledore)");
+        // weak test, no guarantee that we waited long enough for the potential chat window to open
+        await contains(".o-mail-ChatWindow-header:contains(Dumbledore)", 0);
```

Comment:
Shouldn't this crash because of `hasUsedContainsPositively`?

Reply:
click works too